atomic_scanners/container-capabilities-scanner/scanner.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+
+# import docker
+import json
+import logging
+import os
+import subprocess
+import sys
+
+from Atomic import run
+from datetime import datetime
+
+INDIR = "/scanin"
+OUTDIR = "/scanout"
+IMAGE_NAME = os.environ.get("IMAGE_NAME")
+
+# object of atomic run
+run_object = run.Run()
+run_object.image = IMAGE_NAME
+
+# client = docker.Client(base_url="unix:///var/run/docker.sock")
+
+# set up logging
+logger = logging.getLogger("container-pipeline")
+logger.setLevel(logging.DEBUG)
+
+ch = logging.StreamHandler(sys.stdout)
+ch.setLevel(logging.DEBUG)
+formatter = logging.Formatter(
+    '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
```

Comment:
I don't get this one. The link is for this very PR. Have we included lineno in any of the loggers of this repo?